### ISSUE TITLE
Squeeze parameters r.npy for Borealis_M72 invalid?

### ISSUE THREAD

**Issue by leonvarga** [sss441803/BosonSupremacy#2]
Dear authors,

I want to express my appreciation for your insightful publication! I found the idea of separating the quantum part and the classical part to be particularly intriguing.

I have a quick question regarding your code. I attempted to replicate the sampling process for Borealis_M16 and Borealis_M72. While Borealis_M16 worked flawlessly out-of-the-box, I believe including a requirements.txt file would greatly facilitate the setup process. Additionally, it appears that the libraries cvxopt and cvxpy are required for SDP.

I also noticed that the squeezing parameters in r.npy (https://qca-data.s3.amazonaws.com/fig3b/r.npy) for the Borealis_M72 experiment might have been mixed up with some Borealis_M216 parameters. The dimensions of r.npy indicate that it is intended for Borealis_M216 instead of Borealis_M72. Please correct me if I have misunderstood.

Thank you very much for your attention to this matter.

Best regards,
Leon

**Comment by sss441803** [comment]
Hi Leon,

Thank you for the feedback! I  have put `cvxopt` and `cvxpy` as requirements in the readme file. In my experience, when working with HPC environments, versions of packages may have to be tweaked depending on the cluster you are working with, so I have only listed the packages and not the version requirements as I am not clear about that myself.

The download script fetches data from Xanadu's originally provided data location. I just indexed the first 72 entries of the loaded array and that reproduces the exact same covariance matrix we used in our publication. When my colleague did this, he probably confirmed that this is valid. The experiment XEB is sensible with the resulting covariance matrix. Further, the squeezing parameters for Borealis is very uniform since they use a time loop setup with only one source. I have updated the make_cov.py file to make the script work.

Regards,
Henry
